runtime/jcl/common/system.c

```diff
@@ -44,37 +44,6 @@
 #endif
 
 
```

Comment:
Please delete one of the blank lines above (we don't need more than one in a row).